.github/workflows/build-lte.yml

```diff
@@ -64,3 +65,39 @@ jobs:
         with:
           name: LumatoneSetup-macOS
           path: Builds/MacOSX/build/Release
+          if-no-files-found: error
+
+  build-lte-windows:
+    name: Build Lumatone Editor (Windows)
+
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: microsoft/setup-msbuild@v1.1
+      - name: Install JUCE to C:\
+        run: |
+          Invoke-WebRequest https://github.com/juce-framework/JUCE/releases/download/6.1.5/juce-6.1.5-windows.zip -O juce-6.1.5-windows.zip
+          7z x juce-6.1.5-windows.zip -oC:\
+      - name: Build Lumatone Editor
+        run: |
+          bash -c '/c/JUCE/Projucer --resave TerpstraSysEx.jucer'
```

Comment:
If this command is executed in Power Shell it might terminate too early s.t. the builds become flaky.